apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -1631,11 +1632,16 @@ type ClusterResourcePlacementStatus struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// The observed status of ClusterResourcePlacementStatus which mirrors the PlacementStatus of the corresponding ClusterResourcePlacement.
-	// This includes information about the namespace and resources within that namespace that are being managed by the placement.
-	// The status will show placement details for resources selected by the ClusterResourcePlacement's ResourceSelectors.
-	// +kubebuilder:validation:Optional
-	Status PlacementStatus `json:"status,omitempty"`
+	// Replica of the ClusterResourcePlacement status.
+	// +kubebuilder:validation:Required
+	PlacementStatus `json:"statusReplica,omitempty"`
```

Comment:
aks GPT for a better name than "statusReplica"? replica has a special meaning in k8s

Reply:
Changed to sourceStatus instead